tests/e2e/Services/FunctionsScheduler/FunctionsCustomClientTest.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Tests\E2E\Services\FunctionsScheduler;
+
+use Appwrite\Tests\Retry;
+use CURLFile;
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Scopes\SideClient;
+use Tests\E2E\Services\Functions\FunctionsBase;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Role;
+
+class FunctionsCustomClientTest extends Scope
+{
+    use FunctionsBase;
+    use ProjectCustom;
+    use SideClient;
+
+    #[Retry(count: 2)]
```

Comment:
why does this need a retry count? Is this still flaky? If yes we need to understand why

Reply:
Scheduled function sometimes doesn't create execution even after 70s and thus, it is still flaky